runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotConfig.java

```diff
@@ -205,11 +205,23 @@ default boolean screenshotValuableDrop()
 		return false;
 	}
 
+	@ConfigItem(
+		keyName = "valuableDropMinimumValue",
+		name = "Minimum Value for Screenshot",
+		description = "The minimum item value to save a screenshot.",
+		position = 14,
+		section = whatSection
+	)
+	default int valuableDropMinimumValue()
```

Comment:
Instead of adding a separate config here, replace the existing config with this one, allowing `0` to be a value which does not capture screenshots.

Reply:
My only potential concern with this change is that players who currently have valuable drop screenshots on would have them turned off by this change. Then, if they get a valuable drop without realizing the screenshots are now disabled, they might not be too pleased.

But I can make the change if it's what you think is best. Just let me know.